notebooks/scenarios/bigquery/sync/02-configure-api-and-sync.ipynb

```diff
@@ -3,9 +3,37 @@
   {
```

Comment:
<p>in this notebook I would lvoe some headings and structure similar to the current refactored scenario notebooks, its quite hard to read like this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/9281/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>